torchvision/models/swin_transformer.py

```diff
@@ -506,6 +748,75 @@ class Swin_B_Weights(WeightsEnum):
     DEFAULT = IMAGENET1K_V1
 
 
+class Swin_V2_T_Weights(WeightsEnum):
+    IMAGENET1K_V1 = Weights(
+        url="https://download.pytorch.org/models/swin_v2_t-123456.pth",  # TODO
+        transforms=partial(
+            ImageClassification, crop_size=224, resize_size=232, interpolation=InterpolationMode.BICUBIC
+        ),  # TODO
+        meta={
+            **_COMMON_META,
+            "num_params": 28288354,  # TODO
+            "min_size": (224, 224),  # TODO
+            "recipe": "https://github.com/pytorch/vision/tree/main/references/classification#swintransformer",
+            "_metrics": {
+                "ImageNet-1K": {
+                    "acc@1": 0.100,  # TODO
+                    "acc@5": 0.100,  # TODO
+                }
+            },
+            "_docs": """These weights reproduce closely the results of the paper using a similar training recipe.""",
+        },
+    )
+    DEFAULT = IMAGENET1K_V1
```

Comment:
```suggestion
    pass
```

You can add empty weights for now.